requirements.txt

```diff
@@ -0,0 +1,8 @@
+torch==1.3.0
```

Comment:
A specific API in 1.3 version would broke the training process. Maybe you should change it to 0.4=<torch=<1.0